azure-pipelines.yml

```diff
@@ -59,14 +59,18 @@ jobs:
 
   - script: |
       if [[ $NAME == LINT ]]; 
-      then docker run arviz:latest bash -c "./scripts/lint.sh";
+      then docker run -t arviz:latest bash -c "./scripts/lint.sh";
         fi
         
       # Tests saving plots and deleting files
       if [[ $NAME == LINT ]]; 
-      then docker run  --mount type=bind,source="$(pwd)",target=/opt/arviz --name arviz_test arviz:latest bash -c \
+      then docker run  -t --mount type=bind,source="$(pwd)",target=/opt/arviz --name arviz_test arviz:latest bash -c \
       "mkdir -p test_images/new_folder && pytest -v arviz/tests/ --save -k plot_khat --junitxml=junit/test-results.xml --cov=arviz/ --cov-report=xml --cov-report=html";
         fi
+
+      if [ $? -ne 0 ]; then
+        exit 1
+        if
```

Comment:
Shouldn't this be a `fi`? looks like a typo.

Reply:
Yea definitely